packages/common/src/stream.ts

```diff
@@ -224,6 +224,10 @@ export abstract class Stream extends Observable<StreamState> implements StreamSt
     this.state$.next(stream.state)
   }
 
+  async requestAnchor(): Promise<AnchorStatus> {
+    return this.api.requestAnchor(this.id)
+  }
```

Comment:
Should we add the same thing to caip10-link?

Reply:
This is in the Stream base class, so it's available to all StreamType implementations

Reply:
Ah, my bad!